assemble/bin/accumulo

```diff
@@ -59,43 +59,22 @@ function main() {
   export ACCUMULO_CONF_DIR="$conf"
 
   # Verify setting in accumulo-env.sh
-
   : "${JAVA_OPTS:?"variable is not set in accumulo-env.sh"}"
   : "${ACCUMULO_LOG_DIR:?"variable is not set in accumulo-env.sh"}"
   mkdir -p "${ACCUMULO_LOG_DIR}" 2>/dev/null
-  verify_env_dir "ACCUMULO_LOG_DIR" "${ACCUMULO_LOG_DIR}"
   verify_env_dir "HADOOP_PREFIX" "${HADOOP_PREFIX}"
   verify_env_dir "HADOOP_CONF_DIR" "${HADOOP_CONF_DIR}"
   verify_env_dir "ZOOKEEPER_HOME" "${ZOOKEEPER_HOME}"
   : "${MALLOC_ARENA_MAX:?"variable is not set in accumulo-env.sh"}"
 
-  # ACCUMULO_XTRAJARS is where all of the commandline -add items go into for reading by accumulo.
-  # It also holds the JAR run with the jar command and, if possible, any items in the JAR manifest's Class-Path.
+  CLASSPATH="${lib}/accumulo-start.jar:${conf}:${lib}/slf4j-api.jar:${lib}/slf4j-log4j12.jar:${lib}/log4j.jar:${CLASSPATH}"
+
   if [[ "$cmd" = "-add" ]]; then
-    export ACCUMULO_XTRAJARS="$2"
+    CLASSPATH="${2}:${CLASSPATH}"
     shift 2
-  else
-    export ACCUMULO_XTRAJARS=""
   fi
   if [[ "$cmd" = "jar" && -f "$2" ]]; then
-    if [[ $2 =~ ^/ ]]; then
-      jardir="$(dirname "$2")"
-      jarfile="$2"
-    else
-      jardir="$(pwd)"
-      jarfile="${jardir}/${2}"
-    fi
-    if jar tf "$jarfile" | grep -q META-INF/MANIFEST.MF ; then
-      cp="$(unzip -p "$jarfile" META-INF/MANIFEST.MF | grep ^Class-Path: | sed 's/^Class-Path: *//')"
```

Comment:
The only "standard" I would expect is: 1) set classpath, 2) run java.
I think of anything more as "extra". Other projects may have converged on a de facto standard for those "extras", but I'd prefer we just avoid them entirely whenever possible.
It's the "keep it simple, keep it extensible" principle.